org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -735,6 +735,8 @@ GithubOidcAgoraInfraDeploy:
     Repositories:
       - name: "agora2-infra"
         branches: ["main"]
+      - name: "Agora"
```

Comment:
`Sage-Bionetworks/Agora` repo is currently using this role for deployment of the agora app: https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/600-access/_tasks.yaml#L5-L18

We want to replace that role with a more scoped policy. The Agora repo only needs to deploy to beanstalk so i would create a separate resource here with a more scoped (non-admin) policy then delete the one in `600-access folder/_tasks.yaml` folder.

Reply:
OK